test/collector.test.ts

```diff
@@ -232,15 +223,25 @@ describe('Collector', () => {
             expect('updateRemainderAddress').to.be.calledOnContract(collector);
         });
 
-        it('Should update remainder when token balance is a remainder and should withdraw remainder', async () => {
+        it.only('Should update remainder when token balance is a remainder and should withdraw remainder', async () => {
```

Comment:
I think we should remove `.only` from here.
```suggestion
        it('Should update remainder when token balance is a remainder and should withdraw remainder', async () => {
```

Reply:
Fixed, my bad.